client/pages/students/[student_id]/view.tsx

```diff
@@ -0,0 +1,27 @@
+import { trpc } from "@/lib/trpc";
+import { useRouter } from "next/router";
+
+const ViewStudentPage = () => {
```

Comment:
Should this file be called index.ts so a url like `frontend_server/students/<someid>` can use NextJS' file based routing?

Reply:
yeah, doesn't matter too much--was just an example, but can move